### ISSUE TITLE
Add instructions for building on top of this starter

### ISSUE THREAD

**Issue by mtlynch** [ButterCMS/gridsome-starter-buttercms-old#9]
The instructions in the README currently advise the reader to just skip this starter entirely and create a fresh project. If this is a starter project, the user should be able to start from this repo and transition it into their own site.

It's fine if there are additional instructions for starting a fresh repo, but this repo should, at minimum, explain how users can use this starter.

**Comment by jakelumetta** [comment]
@BobWalsh Can you take a look at this one please?

**Comment by jonalecpalustre** [comment]
@mtlynch I made some changes here to be more clear on how to start a gridsome project based on the starter repo. Could you please check this now? - https://github.com/ButterCMS/gridsome-starter-buttercms/blob/master/README.md

Thanks!

**Comment by mtlynch** [comment]
The instructions are still pretty confusing.

>CD into the project and npm install gridsome-source-buttercms to install the ButterCMS gridsome npm module. Let’s also include the npm modules for handling SCSS styling: npm install -D sass-loader node-sass

Why would the user do that instead of just running `npm install`? These packages are included in your `package.json`.

>Moving over to your project code, find your gridsome.config.js file and add to plugins: `{ use: "gridsome-source-buttercms", options: { authToken: 'a985f3f782f2005...,<your AuthToken>', collections: [''], pages: '', pageTypes: '' }`

I think it's simpler to just say something like, "In your gridsome.config.js file, replace the test auth token (`0126997a0f359dffdf0a4200b9c229161c7610ec`) with the auth token for your account."

>Open index.vue in the pages folder (the default view) and make the changes described below in “Change pages/Index.vue”
>...
>Add a new vue component to the components folder, PostCard.vue, using the code below in “Add PostCard.vue” to components.

These seem to be referring to sections that don't exist. Where is the user meant to find these instructions?

**Comment by jonalecpalustre** [comment]
@mtlynch thanks for the feedback! I've updated and make it more simpler. Please check this branch with all the updates - https://github.com/ButterCMS/gridsome-starter-buttercms/blob/ec70b3ca820130b943f8c9a89757cf2f0f57341e/README.md

**Comment by jonalecpalustre** [comment]
@mtlynch  could you please confirm if we are good on this issue too? Or if any feedback needs to be addressed still from our side? Thanks!

**Comment by mtlynch** [comment]
***Note**: The following is a personal comment from me. I don't represent the Gridsome team. You are welcome to submit your starter to Gridsome for review, but I won't be able to approve it personally.*

@jonalecpalustre - Sorry, I have to disengage from this repo. I've filed a few issues to help out ButterCMS, but it's been almost two months to fix issues that should have been resolved in a few hours.

The team has repeatedly asked me to review updates to the repo expecting me to track down changes myself. Most of the time, the changes don't address the issues I've raised.

#7 - Team asks me to review a fix before it's pushed to the demo site (I guess the assumption that I was going to build the repo myself?). Even after the fix, clicking images on the blog index does nothing, which is pretty unusual for a blog.
#8 - Team asks me to review an entire repo instead of the specific commit that fixes it. It took multiple back and forths to address the issue of extraneous READMEs.

#9 (this issue)

We're on revision 3 of these instructions, and it doesn't seem like anyone's given any thought to guiding a user in using this starter. The repo assumes an auth token, but it gives no explanation of how to obtain one.

The current instructions still don't work:

```bash
$ gridsome new my-gridsome-site https://github.com/ButterCMS/gridsome-starter-buttercms
Unknown command new
```

I just took another look at the demo site, and it's now overflowing even on desktop:

![image](https://user-images.githubusercontent.com/7783288/92665121-bd873f00-f2d3-11ea-9ba9-d86cd9296c9f.png)

On mobile, it's even worse.

I just don't feel like the ButterCMS is doing its due diligence here. One or two oversights is understandable, but it just feels like ButterCMS is outsourcing error checking to me as an external reviewer.

**Comment by jonalecpalustre** [comment]
@mtlynch Thank you for your honest feedback here and really sorry this has gone back & forth with some oversights from our end. Really appreciate you sharing your thoughts on how the process went and we'll make sure to learn from this. 

We've fixed the overflowing image issue - https://share.getcloudapp.com/8LuPA7Dj
And have also updated the readme instruction based on your feedback. The Gridsome CLI command has also been updated. 

We'll look into submitting the starter project to Gridsome for review.
